src/metax_api/api/base/api_schemas/catalogrecord.json

```diff
@@ -90,45 +90,74 @@
-                    "type":"string"
+                "editor":{
+                    "title":"Editor",
+                    "description":"The editor that is used to modify the record.",
+                    "type":"object"
                 },
-                "modified_by_api":{
+                "date_modified":{
                     "title":"Date Modified",
                     "description":"Date on which the resource was changed.",
                     "maxItems":1,
                     "type":"string",
                     "format":"date-time",
                     "readonly": true
                 },
-                "modified_by_user_id":{
+                "user_modified":{
                     "title":"Modified by user id",
                     "description":"User id of the person who last edited this record",
                     "maxItems":1,
                     "type":"string"
                 },
-                "created_by_api":{
+                "date_created":{
                     "title":"Date Created",
                     "description":"Date of creation of the resource. Can't be updated once set.",
                     "maxItems":1,
                     "type":"string",
                     "format":"date-time",
                     "readonly": true
                 },
-                "created_by_user_id":{
+                "user_created":{
                     "title":"Created by user id",
                     "description":"User id of the person who created this record. Can't be updated once set.",
                     "maxItems":1,
                     "type":"string"
+                },
+                "service_created":{
+                    "description":"Service name who created this resource.",
+                    "type":"string",
+                    "readonly": true
+                },
+                "service_modified":{
+                    "description":"Service name who last modified this resource.",
+                    "type":"string",
+                    "readonly": true
                 }
             },
             "required":[
                 "data_catalog",
                 "research_dataset"
             ]
+        },
+        "Editor":{
+            "title":"Editor",
+            "description":"An editor object, holds information related to an editor that is used to modify the record. Can contain other fields that are useful to the editor.",
+            "properties":{
+                "owner_id":{
+                    "title":"Owner id",
+                    "description":"Id of the owner in the editor",
+                    "type":"string"
+                },
+                "identifier":{
+                    "title":"identifier",
+                    "description":"Id of the editor, for example: qvain",
+                    "type":"string"
+                }
+            },
+            "required":[
+                "owner_id",
+                "identifier",
+            ]
```

Comment:
According to http://iow.csc.fi/model/mrd/Editor/ , shoudn't this bit have also the creator, or did we agree with Wouter it is the same as the user_created field in the common fields? If so, then iow data model should be changed. Secondly, did we or did we not agree with Wouter the editor object should contain also qvain_id for the record and qvain_modified timestamp?

Reply:
Agreed to add created_id field in another branch to editor objects. All the rest are up to Qvain to add to the catalog record if they wish to do so but they will not be in any way handled in metax nor depicted in iow.

Reply:
Also, Wouter's test owner_id's will be put into place in a different branch.